reporters/kamon-datadog/src/main/scala/kamon/datadog/DatadogSpanReporter.scala

```diff
@@ -101,12 +101,19 @@ class DatadogSpanReporter(@volatile private var configuration: Configuration) ex
   private val logger = LoggerFactory.getLogger(classOf[DatadogSpanReporter])
 
   override def reportSpans(spans: Seq[Span.Finished]): Unit = if (spans.nonEmpty) {
-    val spanList: List[Seq[JsObject]] = spans
+    val spanLists: Array[Array[JsonObject]] = spans
       .map(span => configuration.translator.translate(span, configuration.envTags, configuration.tagFilter).toJson())
-      .groupBy { _.\("trace_id").get.toString() }
+      .groupBy { jsonObj => Option(jsonObj.getNumber("trace_id")).get.toString() }
```

Comment:
this `.get` is gonna always succeed because there's always a `trace_id`, right?

Reply:
Yep that's my understanding. At any rate the logic hasn't changed from what it was previously.